### PR TITLE
Akshita2k

### PR DESCRIPTION
**Pull request by akshita2k**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ import ashwinraj from "../../photos/ashwinraj.jpg";
 import suyashsonawane from "../../photos/suyashsonawane.jpg"
 import harshKumarKhatri from "../../photos/harshKumarKhatri.jpg"
 import AnmolKaur from "../../photos/AnmolKaur.png"
+import akshita2k from "../../photos/akshita2k.png";
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:33]:
```suggestion
import akshita2k from "../../photos/akshita2k.jpg";
```

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -335,8 +336,8 @@ const data = [
         "portfolio": "http://aravindh020.github.io/"
 },
 {
-	"longitude": 76.958896,
-	"latitude": 11.000897,
+	"longitude": 77.620540,
+	"latitude": 13.079790,
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:340]: Did you purposely changed this?

File: src/components/json/worldData.js [hunk 3]
```diff
@@ -524,6 +525,22 @@ const data = [
       "twitter": "https://twitter.com/_AnmolKaur_",
       
     }
+	{
+        "latitude": 28.5192,
+        "longitude": 77.3653,
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:530]: These are my college coordinates 😅

File: src/components/json/worldData.js [hunk 4]
```diff
@@ -524,6 +525,22 @@ const data = [
...
+        "batch": 2020,
+        "photo": akshita2k,
+        "region": "Asia",
+        "skills": ["App Dev", "Web Dev", "ML", "AI", "Data Analysis", "Cloud"],
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:534]: Please select the skills present in the contributions doc only :)

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: src/components/json/worldData.js (hunk 1 above)
Comment (akshita2k) [reply src/components/json/worldData.js]: Done

File: src/components/json/worldData.js (hunk 2 above)
Comment (akshita2k) [reply src/components/json/worldData.js:340]: Yes, just took the copy and edited it

File: src/components/json/worldData.js (hunk 3 above)
Comment (akshita2k) [reply src/components/json/worldData.js]: Changed them, thanks!

File: src/components/json/worldData.js (hunk 4 above)
Comment (akshita2k) [reply src/components/json/worldData.js]: Made the required changes

**Pull request closed without merging**